docker-compose.deps.yml

```diff
@@ -25,10 +25,6 @@ services:
     environment:
       - discovery.type=single-node
 
-  kibana:
```

Comment:
I got rid of Kibana as part of the dev stack. Best to only have it in deployed environments